### ISSUE TITLE
Adding Telekom Mail to Mail app list in Firefox for iOS

### ISSUE THREAD

**Issue by iOS-Martin** [mozilla-mobile/firefox-ios#7828]
### Why/User Benefit/User Problem
I would like to use Firefox as standard browser app on my iOS devices. But the mail app list in Friefox does not list the „Telekom Mal“ app, the mail app of Deutsche Telekom. 

### Acceptance Criteria (how do I know when I’m done?)
Telekom Mail is listed in the list of usable mail apps and, if installed at the device, it can be defined to be used as standard for „mailto“ links.

**Comment by MincDev** [comment]
Is this available to work on?

**Comment by iOS-Martin** [comment]
Yes, in my actual Firefox version 106.1 (20911) the "Telekom mail app" is still not listed.
